Sources/SwiftKuery/Table.swift

```diff
@@ -68,12 +69,18 @@ open class Table: Buildable {
                 if column.isPrimaryKey {
                     columnsWithPrimaryKeyProperty += 1
                 }
-            }
-            else if let label = child.label, label == "tableName" {
+            } else if let label = child.label, label == "tableName" {
```

Comment:
I've thought some more about this and having an explicit check that name is nil makes the code a little more obvious as to where _name gets set. If you want to add it back in it works for me.